config/jobs/kubernetes/sig-release/release-branch-jobs/1.26.yaml

```diff
@@ -75,7 +75,7 @@ periodics:
           cpu: "1"
           memory: 3Gi
 - annotations:
-    fork-per-release-periodic-interval: 6h 24h
+    fork-per-release-periodic-interval: 24h
```

Comment:
This is wrong but something that can be fixed in a follow up (there should be two values).

Reply:
Created #29388 to track this